web/src/components/tasks/search-business-name/SearchBusinessNameForm.tsx

```diff
@@ -47,11 +50,19 @@ export const SearchBusinessNameForm = (props: Props): ReactElement => {
     searchBusinessName,
     updateNameOnProfile,
     resetSearch,
-  } = useBusinessNameSearch({ isBusinessFormation: false, isDba: props.isDba || false });
+  } = useBusinessNameSearch({
+    isBusinessFormation: !!props.isBusinessFormation,
+    isDba: props.isDba || false,
```

Comment:
two things: 
1) why take `isBusinessFormation` - is now ever a time when we are doing a business name search that is NOT formation?
2) I think this is an example of the double-bool thingy I was mentioning after the tech talk. We use 2 bools here (4 states) to represent a 3-state situation. There's no way for isDba to be true when isBusinessFormation is false now, right? Our type model should reflect that